cluster-autoscaler/cloudprovider/azure/azure_config.go

```diff
@@ -248,6 +252,15 @@ func BuildAzureConfig(configReader io.Reader) (*Config, error) {
 			cfg.EnableDynamicInstanceList = dynamicInstanceListDefault
 		}
 
+		if enableVmssFlex := os.Getenv("AZURE_ENABLE_VMSS_FLEX"); enableVmssFlex != "" {
```

Comment:
Could you please point me to the documentation? Maybe I am missing something.
I suggested so because we do use methods I suggested for env variables like "AZURE_ENABLE_DYNAMIC_INSTANCE_LIST"

Reply:
Updated doc as per suggestion